### ISSUE TITLE
Display Mirrored Top to Bottom

### ISSUE THREAD

**Issue by Draco18s** [Microsoft/HoloToolkit-Unity#650]
I usually only see this effect with Image Effects, but when running any of the HoloToolkit example scenes on 5.6, everything is upside down.  *Everything* including the wireframe mesh of the real world.

I tried using a matrix transformation on the camera to flip things around again, but it didn't work.

Not sure what else to try.

**Comment by StephenHodgson** [comment]
Disable MSAA on your camera

**Comment by StephenHodgson** [comment]
Also check out the 5.6 PR https://github.com/Microsoft/HoloToolkit-Unity/pull/618

**Comment by Draco18s** [comment]
Oh ho ho. So THAT'S what's doing it. #LifeSaved 

Project I'm working on (the toolkit and its examples being research thereof) has been having that problem for me *and only me* on this project. My two coworkers (on 5.5) don't have the issue.  Having it crop up in the toolkit examples too (but not my little dickery around project) made me think there was something in common. I did *not* expect it to be a flag on the camera. I can look forward to fewer headaches (and less ibuprofen) in the coming days.

**Comment by StephenHodgson** [comment]
Glad I could help.  I suggest keeping the project versions the same while working collaboratively like that. I'll help keep the cost of ibuprofen down lol.

**Comment by Draco18s** [comment]
Oh yeah, I'd downgrade to 5.5 if I could. I tried about a week ago and ended up with a `XmlDocument could not be found, are you missing an assembly reference?" error that I couldn't resolve. (That was after *two other* errors about nuget packages missing which I had to manually locate and install).  I'm about ready to take my company issued laptop over to IT and tell them to wipe it clean, nuke everything, let me start over.

Other than the issues with 5.6, transferring back and forth hasn't been a problem.
